.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Done. Note that this will automatically modify files, tho

Reply:
Can you elaborate on _"this will automatically modify files"_? I did not understand well the _intent_ behind the remark.

Reply:
Well, the pre-commit bot committed directly to this branch changing the style of the quotations for the `dev` string. It didn't just run the script, it actually committed the changes, which (at least I) do not do usually.

Reply:
Ok I understand now. I thought enabling it would only change the target branch of autoupdate commits, not also enable autofixes.

It seems that maybe we can restore the original behaviour with respect to autofixes using the option `autofix_prs: false`? Check the [pre-commit.ci documentation](https://pre-commit.ci/#configuration). Maybe give it a try by switching the quotes to the old style at the same time that you fix `autofix_prs: false`?

Reply:
We could also consider if we want to leave this activated. Would help if people commit without installing the pre-commit, but I guess could potentially be a risk?

Reply:
I will resolve this therefore and approve the PR, if u want to do something else you're still free to do so.